client/my-sites/plugins/plugin-remove-button/index.jsx

```diff
@@ -57,9 +60,77 @@ module.exports = React.createClass( {
 		}
 	},
 
-	renderButton: function() {
-		var inProgress = PluginsLog.isInProgressAction( this.props.site.ID, this.props.plugin.slug, [ 'REMOVE_PLUGIN' ] );
+	getDisabledInfo: function() {
+		if ( ! this.props.site ) { // we don't have enough info
```

Comment:
we could extract `this.props.site` to `site` and replace the `this.props.site` occurrences